### PR TITLE
make "..." into valid code, say what it represents

### LINKED ISSUE #155: "..." in sound example code blocks can be confusing for new users.
Hey @admsyn hope you're well.  I've got a few students enjoying your sound chapter and as new-ish users translating the abbreviated examples into usable code is proving a bit difficult (the ...) is tripping them up with the inline implementation.  I fixed a few stray typos, which will help a little, but I'm wondering if you would be opposed to being a bit more verbose -- or adding full code examples to the chapter.  Hope you're well and thanks for a great chapter!

### PR DESCRIPTION
**Pull request by admsyn**
The sound chapter included many code blocks which were building on earlier code snippets, and used `...` to represent code that already existed. This isn't valid c++ of course, and was causing some confusion when people tried to use the code verbatim. This PR explains what the ellipses are supposed to represent, and also comments them out to make it valid code.

ping @bakercp, feel free to make any edits you like. I trust your judgement!

Closes #155

### CODE REVIEW

File: chapters/sound/chapter.md [hunk 1]
```diff
@@ -190,22 +190,25 @@ Onset detection algorithms attempt to locate moments in an audio stream where an
 A simple realtime onset detection algorithm can be built on top of the RMS calculation above.
 
 ```cpp
-class ofApp : public ofBaseApp {
-    ...
+class class ofApp : public ofBaseApp {
```
Comment (bakercp) [inline chapters/sound/chapter.md]: class class

**Comment by bakercp** [comment]
This looks like an improvement to me.  Thanks for the updates!

**Comment by mosspassion** [comment]
Cool dudes!

I'm still a little in the dark about how to implement the final example chunk into the overall build that was to be happening during the Synthesizing Audio section. The example right at the end, after the suggestions of further exploration, before the next section "Audio Gotchas" begins. I got the rest of it working! :)

**Comment by admsyn** [comment]
hi @mosspassion, what's tripping you up specifically? Is it the wording or that the code example doesn't work?

**Pull request merged**